src/web/crate_details.rs

```diff
@@ -1105,14 +1256,115 @@ mod tests {
             assert_eq!(platform_links.len(), 2);
 
             for (url, rel) in platform_links {
-                assert!(!url.contains("/target-redirect/"));
-                assert_eq!(rel, "");
+                assert_eq!(
+                    url.contains("/target-redirect/"),
+                    should_contain_redirect,
+                    "ajax: {ajax:?}, should_contain_redirect: {should_contain_redirect:?}",
+                );
+                if !should_contain_redirect {
+                    assert_eq!(rel, "");
+                } else {
+                    assert_eq!(rel, "nofollow");
+                }
             }
+        }
+
+        wrapper(|env| {
+            env.fake_release()
+                .name("dummy")
+                .version("0.4.0")
+                .rustdoc_file("dummy/index.html")
+                .rustdoc_file("x86_64-pc-windows-msvc/dummy/index.html")
+                .rustdoc_file("x86_64-pc-windows-msvc/dummy/struct.A.html")
+                .default_target("x86_64-unknown-linux-gnu")
+                .add_target("x86_64-pc-windows-msvc")
+                .create()?;
+
+            let response = env.frontend().get("/dummy/latest/dummy").send()?;
```

Comment:
We should still have a test for `/crate/dummy/latest` too, making sure that doesn't have redirects.

Reply:
That allowed to find a bug so great idea! :)